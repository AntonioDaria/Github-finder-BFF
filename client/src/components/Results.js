import React from 'react'

const Results = ({ results }) => {
    return (
        <div>
            <h3>Our Best Guess is:</h3>
            <div className="results-container">
                {results}
            </div>
        </div>
    )
}

export default Results
