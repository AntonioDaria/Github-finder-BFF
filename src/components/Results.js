import React from 'react'

const Results = ({results}) => {
    return (
        <>
            <h3>Our Best Guess</h3>
            <ul className="list">
                {results}
            </ul>
        </>
    )
}

export default Results
