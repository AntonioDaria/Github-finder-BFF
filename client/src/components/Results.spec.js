import React from 'react';
import { render } from '@testing-library/react';
import Results from './Results.js'

describe('Results component', () => {
    let container;

    beforeEach(() => {
      ({ container} = renderResult());
    });

    it('should match the snapshot', () => {
        expect(container).toMatchSnapshot();
    });
});

const renderResult = () => {
    return render(
      <Results />
    );
  };