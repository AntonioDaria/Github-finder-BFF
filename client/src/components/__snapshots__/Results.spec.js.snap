// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Results component should match the snapshot 1`] = `
<div>
  <div>
    <h3>
      Our Best Guess is:
    </h3>
    <div
      class="results-container"
    />
  </div>
</div>
`;
